packages/admin/addon/routes/admin/roles.ts

```diff
@@ -4,5 +4,23 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+// @ts-ignore
+import StoreService from '@ember-data/store';
```

Comment:
After upgrading to the latest ember-cli-typescript, can we try:
```suggestion
import type StoreService from '@ember-data/store';
```